lib/env.ts

```diff
@@ -15,6 +15,7 @@ class EnvironmentImpl implements ENV {
   public readonly platform = platform;
 
   public debug = false;
+  public packMode = true;
```

Comment:
>public packMode = true; [](start = 2, length = 23)

just a heads-up: we removed and renamed this boolean. This may result in some merge conflict.

Reply:
fixed